gulpfile.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var gulp = require('gulp');
+var browserSync = require('browser-sync').create();
```

Comment:
Noticed your Jshint config stipulates esversion 6. Hence you should use `const` or `let` for variable declarations instead of `var`.